packages/styles/scss/components/card-link/index.scss

```diff
@@ -52,7 +52,7 @@
     }
 
     &:focus {
-      outline-color: $focus;
+      outline: 2px solid $focus;
```

Comment:
My concern here is if Carbon updates their code to say, `3px` we'd be left in the dust. I opened a ticket with Carbon to get to the root of the problem.

https://github.com/carbon-design-system/carbon/issues/4937

Reply:
Part of me wants to just track in Carbon, but seems like they have a huge backlog of tickets they aren't getting to. Maybe we should just contribute it directly @RaphaelAmadeu @photodow ?